app/PickMe_nebula0/app/src/main/java/com/example/pickme_nebula0/organizer/activities/OrganizerHomeActivity.java

```diff
@@ -91,61 +71,51 @@ public void onPageSelected(int position) {
     }
 
     /**
-     * Configures the back button to navigate to the previous activity.
-     *
-     * Attaches a click listener to the back button that triggers the system's back press dispatcher.
+     * Function for back button
      */
     private void backButton() {
         Button backButton = findViewById(R.id.backButton);
-        backButton.setOnClickListener(v -> getOnBackPressedDispatcher().onBackPressed());
-    }
+        Animation buttonClickAnimation = AnimationUtils.loadAnimation(this, R.anim.button_click_animation);
 
-    private void refreshButton(){
```

Comment:
this goes along with my other comment about the refresh button, this should not have been deleted